src/components/header/style.module.css

```diff
@@ -94,11 +96,7 @@
 	}
 
 	nav {
-		position: relative;
-		display: inline-block;
-		font-size: 100%;
-		user-select: none;
-		-webkit-user-drag: none;
+		display: flex;
```

Comment:
Fixes the issue of header items wrapping on some screensizes:

![Screenshot of the site at a width of 900px. The header nav items wrap so that the 'REPL' link is pushed out of the header and on top of the page content](https://github.com/user-attachments/assets/d9a030bf-b564-486b-b29d-a8c840354775)

Still not perfect, there's now some overlapping in the header, but wrapping is very bad.